lib/helper/Playwright.js

```diff
@@ -1573,15 +1577,9 @@ class Playwright extends Helper {
     const matcher = await this.context;
     let waiter;
     const context = await this._getContext();
-    if (locator.isCSS()) {
-      const enabledFn = function (locator) {
-        const els = document.querySelectorAll(locator);
-        if (!els || els.length === 0) {
-          return false;
-        }
-        return Array.prototype.filter.call(els, el => !el.disabled).length > 0;
-      };
-      waiter = context.waitForFunction(enabledFn, { timeout: waitTimeout }, locator.value);
+    if (!locator.isXPath()) {
```

Comment:
Using combined selectors in playwright to 
1. find matching element using custom selector
2. filter the found elements down to those that are not disabled